great_expectations/execution_engine/pandas_execution_engine.py

```diff
@@ -444,11 +444,13 @@ def _get_reader_fn(self, reader_method=None, path=None):
                 f'Unable to find reader_method "{reader_method}" in pandas.'
             )
 
-    def resolve_metric_bundle(  # type: ignore[empty-body]
+    def resolve_metric_bundle(
         self, metric_fn_bundle
```

Comment:
@alexsherstinsky can you add an annotation (other than `Any`) for `metric_fn_bundle`?

Reply:
@Kilo59 No -- this has already been discussed before -- it can be `Callable`, any of a great many of `SqlAlchemy` functions, and any of a great many `Spark` functions.  I do not see an easy possibility here.  Happy to discuss.  Thanks.

Reply:
`Callable` is better than no annotation.